### ISSUE TITLE
Attempting to Install Tensorflow

### ISSUE THREAD

**Issue by Dyn-Walker** [Anaconda-Platform/anaconda-client#631]
I have made repeated attempts to install tensorflow, all of which all failed. I tried installing it through Anaconda Navigator >Environments, toggling both Keras and Tensorflow. The computer just spins for hours with a blue bar cycling at the bottom of the "Install Packages" window "Solving package specifications" - no success.

I have multiple attempts using the Anaconda Navigator Command line using "conda install -c conda-forge tensorflow", then "conda install -c "conda-forge/label/broken" tensorflow", as suggested on the web-page https://anaconda.org/conda-forge/tensorflow. The computer spends hours cycling through almost a hundred models, then spends another couple of hours "Resolving Conflicts" with the final result - Installation 0. 

The installation either fails or does not proceed with the installation, essentially the same thing.

How can I install Tensorflow, then Keras?
